docs/source/install.rst

```diff
@@ -0,0 +1,570 @@
+           "s3:GetObject",
+           "s3:DeleteObject"
+         ],
+         "Resource": "arn:aws:s3:::${S3_BUCKET_NAME}/*"
+       }
+     ]
+   }
+   EOF
+
+   export LLM_OPERATOR_POLICY="LLMOperatorPolicy"
+   aws iam create-policy --policy-name "${LLM_OPERATOR_POLICY}" --policy-document file://policy.json
+
+   export LLM_OPERATOR_SERVICR_ACCOUNT_NAME="llm-operator"
+   eksctl create iamserviceaccount \
+     --name "${LLM_OPERATOR_SERVICR_ACCOUNT_NAME}" \
+     --namespace "${LLM_OPERATOR_NAMESPACE}" \
+     --cluster "${CLUSTER_NAME}" \
+     --role-name "LLMOperatorRole" \
+     --attach-policy-arn "arn:aws:iam::${AWS_ACCOUNT_ID}:policy/${LLM_OPERATOR_POLICY}" --approve
+
+
+Step 7. Install Milvus
+^^^^^^^^^^^^^^^^^^^^^^
+
+TODO(kenji): Fill this out.
+
+
+Step 8. Install LLM Operator
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+.. code-block:: console
+
+   # Set the endpoint URL of LLM Operator. Please change if you are using a different ingress controller.
+   export INGRESS_CONTROLLER_URL=http://$(kubectl get services -n kong kong-proxy-kong-proxy  -o jsonpath='{.status.loadBalancer.ingress[0].hostname}')
+
+   cat << EOF | envsubst > llm-operator-values.yaml
+   global:
+     database:
+       host: "${POSTGRES_ADDR}"
+       port: ${POSTGRES_PORT}
+       username: "${POSTGRES_USER}"
+       ssl:
+         mode: require
+       createDatabase: true
+
+     databaseSecret:
+       name: "${POSTGRES_SECRET_NAME}"
+       key: password
+
+     objectStore:
+       s3:
+         bucket: "${S3_BUCKET_NAME}"
+         region: "${S3_REGION}"
+
+     # This is an ingress configuration with Kong. Please change if you are using a different ingress controller.
+     ingress:
+       ingressClassName: kong
+       # The URL of the ingress controller. this can be a port-forwarding URL (e.g., http://localhost:8080) if there is
+       # no URL that is reachable from the outside of the EKS cluster.
+       controllerUrl: "${INGRESS_CONTROLLER_URL}"
+       annotations:
+         # To remove the buffering from the streaming output of chat completion.
+         konghq.com/response-buffering: "false"
+
+   inference-manager-engine:
+     serviceAccount:
+       create: false
+       name: "${LLM_OPERATOR_SERVICE_ACCOUNT_NAME}"
+     model:
+       default:
+         runtimeName: vllm
+         resources:
+           limits:
+             nvidia.com/gpu: 1
+       overrides:
+         meta-llama/Meta-Llama-3.1-8B-Instruct-q4_0:
+           preloaded: true
+           contextLength: 16384
+         google/gemma-2b-it-q4_0:
+           runtimeName: ollama
+           preloaded: true
+           resources:
+            limits:
+              nvidia.com/gpu: 0
+         sentence-transformers/all-MiniLM-L6-v2-f16:
+           runtimeName: ollama
+           preloaded: true
+           resources:
+            limits:
+              nvidia.com/gpu: 0
+
+   inference-manager-server:
+     service:
+       annotations:
+         konghq.com/connect-timeout: "360000"
```

Comment:
might want to add a comment that these are needed only if using `kong` for ingress?

Reply:
done